doc/ref_model/chapters/chapter05.md

```diff
@@ -100,7 +100,7 @@ The following sections detail the NFVI SW profile features per type of virtual r
 
 | .conf | Feature | Type  | Description |
 |------------------|----------------|----------------|------------------------------------------------------------------------------------------------|
-| nfvi.net.acc.cfg.001 | vSwitch optimisation | Yes/No and SW Optimisation | e.g. DPDK. |
+| nfvi.net.acc.cfg.001 | vSwitch optimisation | Yes/No and SW Optimisation | e.g. DPDK. <br> **Note:** This feature is not applicable for Kubernetes based infrastructures due to lack of vSwitchm however workloads need access to user space networking solutions. |
```

Comment:
@CsatariGergely The qualification part of the line sounds like RA.

> however workloads need access to user space networking solutions.
Does this gap need to be a new Issue?

Reply:
I do not think so, in case of conteiners there is no vSwitch, but user space networking technologies are still available.